### ISSUE TITLE
For some reason Plato doesn't follow internal links of some PDFs

### ISSUE THREAD

**Issue by Masacroso** [baskerville/plato#296]
I noticed this behavior in at least the last two versions of Plato used in Kobo Forma (last two versions counted from today), I press a link of a PDF and it does nothing instead of going to the corresponding page of the book.

The PDF is [here](https://file.io/pLV75njiXjg8), if someone want to test it.

**Comment by Masacroso** [comment]
Sorry sir, but the bug persists. I build Plato from source code 30 min ago, put the builded version on my Kobo Forma (deleting the old one) and the same behavior persists.

**Comment by baskerville** [comment]
> Sorry sir, but the bug persists.

I don't think so. If you had successfully replaced the *plato* binary, then it would have crashed during startup, because the _MuPDF_ third-party dependency was updated since 0.9.36.

**Comment by Masacroso** [comment]
> > Sorry sir, but the bug persists.
> 
> I don't think so. If you had successfully replaced the _plato_ binary, then it would have crashed during startup, because the _MuPDF_ third-party dependency was updated since 0.9.36.
> 
> **EDIT**: It would also just fail during the build phase because of the _MuPDF_ version mismatch.

I just build everything with ./build.sh and after I did ./dist.sh as shown in the [build guide](https://github.com/baskerville/plato/blob/master/doc/BUILD.md). I did all steps correctly with no errors, giving a zip named plato0.36 (or something like, I dont remember exactly the name of the zip file). I uncompressed this zip in a plato folder, and substitute (deleted the old one and put this) the plato folder in kobo forma with this one. It runs perfectly, with no crash at all but the bug persists.

Maybe I must just substitute the plato binary in the old installation?
